src/shared/actions/alerts.js

```diff
@@ -231,6 +231,23 @@ export const generateAccountSyncRetryAlert = () => (dispatch) =>
         ),
     );
 
+/**
+ * Generates an info alert (if addresses syncing fails) for auto retrying addresses sync
+ *
+ * @method generateAddressesSyncRetryAlert
+ *
+ * @returns {function} dispatch
+ */
+export const generateAddressesSyncRetryAlert = () => (dispatch) =>
+    dispatch(
+        generateAlert(
+            'info',
+            i18next.t('global:pleaseWait'),
+            `${i18next.t('global:errorSyncingAddresses')} ${i18next.t('global:tryingAgainWithDifferentNode')}`,
```

Comment:
It might be best to make a new string for this instead of combining two existing ones

Reply:
All right. Will update.

Reply:
@rajivshah3 Addresses in https://github.com/iotaledger/trinity-wallet/pull/588/commits/443370cb11d058e7dc9e7947957fa7ea38df27a6